index.js

```diff
@@ -26,11 +27,15 @@ function constructRequest(har) {
     querystring = `?${query}`;
   }
 
+  if (userAgent) {
+    options.headers['User-Agent'] = userAgent;
+  }
+
   return new Request(`${url}${querystring}`, options);
 }
 
-function fetchHar(har) {
-  return fetch(constructRequest(har));
+function fetchHar(har, userAgent) {
```

Comment:
Wouldn't the HAR have the user agent information in there already? Maybe I am misunderstanding the intent here

Reply:
Since we don't want the "ReadMe API Explorer" user agent to show up in the code samples, we shouldn't modify the HAR.

Reply:
Got it.

Reply:
Got it.